test/functionalTest/cases/1029_get_many_entity_types/get_many_entity_types.test

```diff
@@ -0,0 +1,2295 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Move this file to the proper place. The cases/1029 doesn't seem the right place (issue https://github.com/telefonicaid/fiware-orion/pull/1029 has nothing to do with this...)

(Self-note)

Reply:
Fixed in 663c53492c1c79a62818c2ba86e7dc6111e4cf97